console/src/components/Main/Main.js

```diff
@@ -105,12 +91,6 @@ class Main extends React.Component {
     }
   }
 
```

Comment:
I think we need to `fetchServerConfig` for JWT decoder.

Reply:
Yep. I moved it to line 69. Removed unnecessary fn